radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/PersistenceModule.java

```diff
@@ -115,7 +115,7 @@ StoreConfig storeConfig() {
 		return update -> {
 			systemCounters.increment(CounterType.PERSISTENCE_VERTEX_STORE_SAVES);
 			persistentVertexStore.save(update.getVertexStoreState());
-		};
+		};	//TODO: DISPATCH: processing PersistentVertexStore may throw BerkeleyStoreException
```

Comment:
`BerkeleyStoreException` is a true `panic` exception so it shouldn't be handled.